apps/app-article/src/App.js

```diff
@@ -95,12 +84,37 @@ class App extends Component {
     componentWillUnmount() {
     }  
     onLogout() {
-        Android.onLogoutSuccess();
+        //To avoid undefined error for ios webview
+        try {
+            Android.onLogoutSuccess();
+        } catch (e) {
+            console.log(e);
```

Comment:
If the purpose is to debug, we could use `console.debug(e)` instead of `log`.

Reply:
@vapaj Yes it can be replaced, should i send new commit ?

Reply:
I mean, the log will stay in the browser so it's not much use.

Reply:
Actually it could be replaced with Android == 'undefined', because the reason of that try{}catch just to avoid error log that could prevent some phone version from rendering articles 
@vapaj